src/vivarium_public_health/population/data_transformations.py

```diff
@@ -123,12 +123,11 @@ def rescale_binned_proportions(
         "value",
     ]
     for _, sub_pop in pop_data.groupby(["sex", "location"]):
-
         min_bin = sub_pop[(sub_pop.age_start <= age_start) & (age_start < sub_pop.age_end)]
-        padding_bin = sub_pop[sub_pop.age_end == float(min_bin.age_start)]
+        padding_bin = sub_pop[sub_pop.age_end == float(min_bin.age_start.iloc[0])]
```

Comment:
Nit: We generally prefer using bracket notation rather than dot notation. This seems like a good excuse to clean that up in this function.

Reply:
Good call. I'll update.